vital/bindings/python/Kwiver_Python_Demo.ipynb

```diff
@@ -1,606 +0,0 @@
-{
```

Comment:
I don't remember exactly what was in this notebook, but it might be worth while adapting what may have previously been in here to the current bindings as a quick-start or otherwise example usage of something.

Reply:
It was an introduction to c based python bindings and algorithms

Reply:
Noted in #878 and would be addressed once kwiver wheel is complete